README.md

```diff
@@ -1,2 +1,39 @@
-# cloudflare-worker-template-typescript
-Netacea Wrangler TypeScript project
+# Netacea Cloudflare Worker Template
+![Netacea Header](./header.jpg)
+[![TypeScript](https://img.shields.io/badge/%3C%2F%3E-TypeScript-%230074c1.svg)](http://www.typescriptlang.org/)
+
+A simple Cloudflare worker with Netacea built in.
+
+## 💡 Getting Started
+
+This template is meant to be used with [Wrangler](https://github.com/cloudflare/wrangler). If you are not already familiar with the tool, we recommend that you install the tool and configure it to work with your [Cloudflare account](https://dash.cloudflare.com). Documentation can be found [here](https://developers.cloudflare.com/workers/tooling/wrangler/).
+
+To generate using Wrangler, run this command:
+
+```bash
+wrangler generate netacea-cloudflare-worker https://github.com/netacea/cloudflare-worker-template-typescript
+```
+
+Ensure the `@netacea/cloudflare` package is up to date by running:
+```bash
+npm i @netacea/cloudflare@latest
+```
+
+Insert your Netacea API and Secret key into `./src/NetaceaConfig.json`. Ensure you are [logged into wrangler](https://github.com/cloudflare/wrangler#-login). Insert your wrangler credentials into the `wrangler.toml` file in the root directory (you only need to change `account_id`, `route` and `zone_id` - the rest is okay).
```

Comment:
I would convert this body of text into 3 bullet points so users are more likely to read/scan over the steps required

Reply:
Done! Nice one